src/main/clojure/clara/rules/durability.clj

```diff
@@ -401,60 +401,87 @@
                 :activation-group-fn
                 :alphas-fn))))
 
-(defn create-default-get-alphas-fn [rulebase]
-  (@#'com/create-get-alphas-fn type ancestors rulebase))
+(def ^:private create-get-alphas-fn @#'com/create-get-alphas-fn)
+
+(defn- opts->get-alphas-fn [rulebase opts]
```

Comment:
I don't think this is quite right either, and is unnecessarily complex.  How about

````
(defn opts->get-alphas-fn [rulebase opts]
       (let [fact-type-fn (or (:fact-type-fn opts) type)
              ancestors-fn (or (:ancestors-fn opts) ancestors)]
       (create-get-alphas-fn fact-type-fn ancestors-fn rulebase)))
````

Currently this doesn't allow the user to specify the ancestors-fn without overriding the fact-type-fn as well.  More generally, I don't think we need to intermingle the "default or override" selection logic with the call to actually create the get-alphas-fn; doing so just creates unnecessary complexity.